docs/installation.rst

```diff
@@ -0,0 +1,51 @@
+============
```

Comment:
```suggestion
:orphan:

============
```

Reply:
this is already sovled by adding the new files into a the strucure and into the hidden toctrees in main.